delfin/drivers/utils/ssh_client.py

```diff
@@ -132,6 +132,7 @@ def do_exec(self, command_str):
 
 class SSHPool(pools.Pool):
     SOCKET_TIMEOUT = 10
+    MAX_POOL_SIZE = 3
```

Comment:
This is common file, may be it can be 5, please check.

Reply:
done